projects/plugins/jetpack/extensions/blocks/ai-chat/block.json

```diff
@@ -3,7 +3,7 @@
 	"apiVersion": 3,
 	"name": "jetpack/ai-chat",
 	"title": "AI Chat",
-	"description": "Provides summarized chat across a site’s content, powered by AI magic.",
+	"description": "Provide summarized chat across a site’s content, powered by AI magic.",
```

Comment:
Hi @monsieur-z.
This line has been [edited in trunk](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/blocks/ai-chat/block.json#L6), please take a look.